pkg/analyze/host_subnetavailable.go

```diff
@@ -0,0 +1,82 @@
+package analyzer
+
+import (
+	"encoding/json"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/replicatedhq/troubleshoot/pkg/collect"
+)
+
+type AnalyzeHostSubnetAvailable struct {
+	hostAnalyzer *troubleshootv1beta2.SubnetAvailableAnalyze
+}
+
+func (a *AnalyzeHostSubnetAvailable) Title() string {
+	return hostAnalyzerTitleOrDefault(a.hostAnalyzer.AnalyzeMeta, "Subnet Available")
+}
+
+func (a *AnalyzeHostSubnetAvailable) IsExcluded() (bool, error) {
+	return isExcluded(a.hostAnalyzer.Exclude)
+}
+
+func (a *AnalyzeHostSubnetAvailable) Analyze(getCollectedFileContents func(string) ([]byte, error)) ([]*AnalyzeResult, error) {
+	hostAnalyzer := a.hostAnalyzer
+
+	name := filepath.Join("host-collectors/subnetAvailable", "result.json")
+	if hostAnalyzer.CollectorName != "" {
+		name = filepath.Join("host-collectors/subnetAvailable", hostAnalyzer.CollectorName+".json")
```

Comment:
This too

Reply:
see https://github.com/replicatedhq/troubleshoot/pull/1004#discussion_r1113765803

Reply:
`wontfix`, as per https://github.com/replicatedhq/troubleshoot/pull/1004#discussion_r1113765803